src/lib/features/personal-dashboard/personal-dashboard-read-model.ts

```diff
@@ -80,6 +80,7 @@ export class PersonalDashboardReadModel implements IPersonalDashboardReadModel {
             (project: IntermediateProjectResult) => {
                 const roles = Object.values(project.roles);
                 roles.sort((a, b) => a.id - b.id);
+
```

Comment:
not sure what happened on this line

Reply:
probably a stray console.log that i added and then removed
```suggestion
```